apprentice_teacher.rb

```diff
@@ -1,12 +1,18 @@
-class ApprenticeTeacher
+require_relative "teachers"
+
+class ApprenticeTeacher < Teachers
   attr_reader :age, :salary, :phase, :target_raise
   attr_accessor :name
 
+  TARGET_RAISE = 800
+  PHASE = 3
+
   def initialize(options={})
-    @age = options.fetch(:age, 0)
-    @name = options.fetch(:name, "")
-    @target_raise = 800
-    @phase = 3
+    # @age = options.fetch(:age, 0)
+    # @name = options.fetch(:name, "")
+    super
+    # @target_raise = 800
+    # @phase = 3
   end
```

Comment:
If the only thing you're doing in your method is call up to super, might as well remove the method from your class altogether.